src/include/distributed/resource_lock.h

```diff
@@ -110,6 +110,10 @@ typedef enum CitusOperations
 						 ADV_LOCKTAG_CLASS_CITUS_PLACEMENT_CLEANUP)
 
 
+#define LOCK_RELATION_IF_EXISTS "SELECT lock_relation_if_exists(%s, '%s');"
+#define LOCK_RELATION_IF_EXISTS_NOWAIT \
+	"SELECT lock_relation_if_exists(%s, '%s', nowait => true);"
```

Comment:
while we're moving these, it would be good to include `pg_catalog.` to avoid search_path overloading issues.